AEPIdentity/Sources/MobileIdentities.swift

```diff
@@ -127,13 +127,13 @@ struct MobileIdentities: Codable {
 
         return CompanyContext(marketingCloudId: marketingCloudOrgId)
     }
-}
 
-private extension CustomIdentity {
-    /// Converts a `CustomIdentity` into a `UserID`
+    /// Converts a `CustomIdentity` dictionary representation into a `UserID`
     /// - Returns: a `UserID` where the namespace is value, value is identifier, and type is integrationCode
-    func toUserID() -> UserID? {
-        guard let type = type, let identifier = identifier, !identifier.isEmpty else { return nil }
+    private func dictToUserID(dict: [String: Any]) -> UserID? {
```

Comment:
can this be a static method?

Reply:
I suppose it could be, what's the use case?

Reply:
just easier to understand the behavior of the method, it won't change any instance vars